pyproject.toml

```diff
@@ -49,13 +49,13 @@ classifiers = [
 casa = ["python-casacore>=3.5.2"]
 healpix = ["astropy-healpix>=1.0.2"]
 moon = ["lunarsky>=0.2.5"]
-sim = ["mpi4py>=3.1.1"]
+sim = ["mpi4py>=3.1.1", "psutil"]
 all = ["pyuvsim[casa,healpix,moon,sim]"]
 test = ["coverage", "pre-commit", "pytest", "pytest-cov>=5.0"]
 doc = ["matplotlib", "pypandoc", "sphinx"]
 profiler = ["line-profiler"]
 dev = ["pyuvsim[all,test,doc,profiler]"]
-
+windows-dev = ["pyuvsim[test,doc,profiler,healpix,moon,sim]"]
```

Comment:
I think we need to document this new option in the readme.

We also need some more general docs updates to make it clear that pyuvsim works on Windows.

Reply:
OK, I added:

* Trove classifiers to the pyproject.toml to make it clear that windows is supported
* Several sentences to the readme (in difference sections) to make it clear that windows is (partially) supported, and instruct on how to install for a dev environment on windows.